Implementado consumo de feed do Medium

leonardo-lemos opened the pull request:
Implementado consumo de feed do Medium da comunidade através de nosso API Gateway

PabloDinella reviewed (approved):
Podemos melhorar bastante o código e aplicar algum guia de estilo (o elementary oficial tem um não tem?), mas podemos fazer isso numa futura refatoração ;)

leonardo-lemos committed:
Implementado consumo de feed do Medium

Implementado consumo de feed do Medium da comunidade através de nosso API Gateway
